### PR TITLE
Grab po files for the docs.

### PR DESCRIPTION
**Pull request by MCGallaspy**
**What's inside:**
*  Remove some `import_all_child_modules` vestiges.
*  Update the `update_pot` command to grab messages from the docs, in addition to the app.

The submodule needs to incorporate changes from [this branch](https://github.com/MCGallaspy/ka-lite/tree/ss-translation) in order for this to work, and furthermore before you run this command you need to first grab the dependencies from `sphinx-docs/requirements.txt` in the submodule.

### CODE REVIEW

File: centralserver/i18n/management/commands/update_pot.py [hunk 1]
```diff
@@ -91,10 +91,23 @@ def inside_kalite():
...
         ensure_dir("locale")
         call_command("makemessages", locale="en", ignore_patterns=ignore_patterns, no_obsolete=True, domain="django")
         call_command("makemessages", locale="en", ignore_patterns=ignore_patterns, no_obsolete=True, domain="djangojs")
-        return glob.glob(os.path.join(''.join(settings.LOCALE_PATHS), "en", "LC_MESSAGES", "*.po"))
+        pofiles = glob.glob(os.path.join(''.join(settings.LOCALE_PATHS), "en", "LC_MESSAGES", "*.po"))
+        # Then for the docs
+        os.chdir(os.path.join("ka-lite-submodule", "sphinx-docs"))
+        p = Popen(["make", "gettext"])
+        p.wait()
+        p = Popen(["sphinx-intl", "update", "-p", "_build/locale", "-l", "en"])
```
Comment (aronasorman) [inline centralserver/i18n/management/commands/update_pot.py]: The central server uses `python2.7`, so it's preferred to use `subprocess.call` in here.

**Comment by aronasorman** [comment]
Aside from one comment, changes look good.

**Comment by MCGallaspy** [comment]
Submodule should be updated to include changes from [this PR](https://github.com/learningequality/ka-lite/pull/3480), then this can be merged.

**Comment by MCGallaspy** [comment]
Merge conflicts? :(

**Comment by aronasorman** [comment]
Rebase!

File: centralserver/i18n/management/commands/update_pot.py [hunk 2]
```diff
@@ -36,7 +36,7 @@
 CROWDIN_API_URL = "https://api.crowdin.com/api/project"
 
 
-class Command(test_wrappings.Command):
+class Command(BaseCommand):
```
Comment (aronasorman) [inline centralserver/i18n/management/commands/update_pot.py]: This thing has been fixed in 3 branches so far. Ha!

**Pull request merged**